### ISSUE TITLE
Add a simpler way to run an individual spec file

### ISSUE THREAD

**Issue by homeofmatt** [roberts1000/rspec_n#20]
I've been able to run individual specs by just modifying the `--command` arg to be like: `bundle exec rspec spec/path/to/test.rb`, but considering how often specific spec files/directories are run, it would be nice to add an optional direct argument (like rspec) to just specify a file/directory
Possible usage ex:
```
rspec_n 5 spec/path/to/test.rb --command 'bundle exec rspec'
```
Or `rspec_n 5 spec/path/to/tests/*`

**Comment by roberts1000** [comment]
Good idea!

**Comment by roberts1000** [comment]
@homeofmatt Thanks for the feedback.  I just released v1.1.0 which has this enhancement.
